pkg/daemon/update.go

```diff
@@ -226,17 +236,27 @@ func (dn *Daemon) enableUnit(unit ignv2_2types.Unit) error {
 	}
 	// The originating file to link
 	servicePath := filepath.Join(pathSystemd, unit.Name)
-	return os.Symlink(servicePath, wantsPath)
+	err := os.Symlink(servicePath, wantsPath)
+	if err != nil {
+		glog.Warningf("Cannot enable unit %s: %s", unit.Name, err)
+	} else {
+		glog.Infof("Enabled %s", unit.Name)
+		glog.V(2).Infof("Symlinked %s to %s", servicePath, wantsPath)
+	}
+    return err
```

Comment:
nit: `gofmt`. this line has spaces instead of tabs, and there are a couple of other whitespace issues in the file as well.

Reply:
Done.